contrib/aktualizr-toml-update

```diff
@@ -5,4 +5,13 @@
 
 [ -z "$CONFIG_FILE" ] && (echo "No CONFIG_FILE specified"; exit 1)
 cp $CONFIG_FILE /etc/sota/conf.d/
+
+# If we take control of tags/apps, then we should try and remove from sota.toml
+if grep -q ^tags $CONFIG_FILE 2>/dev/null ; then
+	sed -e '/^tags/ s/^#*/# MANAGED BY FIOCONFIG: /' -i /var/sota/sota.toml
+fi
+
+if grep -q ^compose_apps $CONFIG_FILE 2>/dev/null ; then
+	sed -e '/^compose_apps/ s/^#*/# MANAGED BY FIOCONFIG: /' -i /var/sota/sota.toml
```

Comment:
This reg exp changes the` compose_apps_root` param in sota.toml too, I am not sure if there is an intention to do so?

Reply:
nope. i should make that explicit. good catch

Reply:
Yeah, better to go for a full line match to avoid future issues as well.

Reply:
thanks for the feedback. force-push fixed